### PR TITLE
Supporting search for DMS with padded zeros

### PR DESCRIPTION
**Pull request by procrastinatio**
Hopefully fixing https://github.com/geoadmin/mf-geoadmin3/issues/4051

<jenkins>[Test link](https://mf-geoadmin3.int.bgdi.ch/mom_skyguide/index.html)</jenkins>

### CODE REVIEW

File: test/specs/search/SearchService.spec.js [hunk 1]
```diff
@@ -228,7 +228,9 @@ describe('ga_search_service', function() {
         '46°58′16.320030760136433" N  6°58′10.12802667678261" E',
         '46º58′16.320030760136433″ N  6º58′10.12802667678261″ E',
         '46° 58\' 16.320030760136433\'\' N 6° 58\' 10.12802667678261\'\' E',
-        // Quandrants are wrongi/missing, but we assume in Switzerland
+        // Skyguide special (zero padded)
+        '46°58\'16.320030760136433" N  006°58\'10.12802667678261" E',
```
Comment (oterral) [inline test/specs/search/SearchService.spec.js]: Add also a test with 1 zero.

File: test/specs/search/SearchService.spec.js [hunk 2]
```diff
@@ -248,6 +250,15 @@ describe('ga_search_service', function() {
           $rootScope.$digest();
         });
       });
+
+      it('trying to parse Skyguide format (exact string)', function(done) {
```
Comment (oterral) [inline test/specs/search/SearchService.spec.js]: This test is not necassary since you already add the case l231

Comment (oterral) [reply test/specs/search/SearchService.spec.js]: ah ok you test DMS here forget the previous comment.

File: test/specs/search/SearchService.spec.js (hunk 1 above)
Comment (oterral) [reply test/specs/search/SearchService.spec.js]: I've adde dit

Comment (procrastinatio) [reply test/specs/search/SearchService.spec.js]: @oterral OK, I see. Switzerland spaning betwen 005°E and 011°E
